libs/template/experimental/virtual-scrolling/src/lib/model.ts

```diff
@@ -34,6 +34,17 @@ export interface ListRange {
   end: number;
 }
 
+export abstract class DataSource<T> {
```

Comment:
do you know if it's possible to maybe have an `import type { DataSource } from '@angular/cdk'` without having it to mention as `peerDependency`?

That would be the absolute best case